build/signedbuild.yml

```diff
@@ -11,6 +11,14 @@ variables:
   FAKES_SUPPORTED: 1
 
 jobs:
```

Comment:
I think the SignedRelease job needs these jobs under `dependsOn`

Reply:
Good catch, updated